test/utils/cryptography/P256.t.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {P256} from "@openzeppelin/contracts/utils/cryptography/P256.sol";
+import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
+
+contract P256Test is Test {
+    /// forge-config: default.fuzz.runs = 512
+    function testVerify(uint256 seed, bytes32 digest) public {
+        uint256 privateKey = bound(uint256(keccak256(abi.encode(seed))), 1, P256.N - 1);
```

Comment:
this is cheaper and faster:
```suggestion
    function testVerify(string calldata seed, bytes32 digest) public {
        (, uint256 privateKey) = makeAddrAndKey(seed);
```

also for `testRecover`

Reply:
Good catch, though the difference here is that the order of G (N) is way smaller for P256 than for secp256k1 (which I assume is what `makeAddrAndKey` uses). Perhaps it's enough with `privateKey = bound(privateKey, 1, P256.N - 1)`

Reply:
yes good point, i think we can do `privateKey = bound(privateKey, 1, P256.N - 1)`

Reply:
added in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4881/commits/cbd2ff5b16a47c46381626c82cafc6221d98aa54